### DISCUSSION TITLE
Laptop fans constantly spin on high if i don't have fan control open

### DISCUSSION THREAD

**Discussion by AdamXBlack** [T-Troll/alienfx-tools#210] [Q&A]
I need help with this, I recently couldn't get AWCC to install anymore so I downloaded this alternative to be able to control my lights.  I accidentally clicked on the fan control and it automatically installed it and now my fans will not stop spinning at max speed unless I have fan control open.  How can I completely get rid of everything and get back to normal?

**Comment by T-Troll** [comment]
Hmm... Fan control remember the power mode it initiated and restore it at the application exit, so seems like you BIOS switched into "Full speed" state.
In fact, Power Modes at Fans tab the same as into AWCC, just have different names and have manual mode.
What to do: you can disable "Fan control" at Settings tab if you don't need it (but i don't reccomend, better set desired power mode (and you can switch it any time using keyboard shortcuts).
Or, you can just alter default system mode using CLI application - run `alienfan-cli setpower=N` from Administrator CMD, there N is desired power mode (0 for manual, the rest is depended on your system. NB: GUI app should be stopped first to do this!
